docs/src/components/DocsLayout.tsx

```diff
@@ -86,19 +86,17 @@ class DocsLayout extends React.Component<any, any> {
             },
           })}
```

Comment:
These styles were previously use to style `MenuItem`. I think it's cruft now...

We can move `treeItemStyles` there...